scripts/qa_crowbarsetup.sh

```diff
@@ -152,6 +152,8 @@ function mount_localreposdir_target
     fi
     mkdir -p $localreposdir_target
     if ! grep -q "$localreposdir_target\s\+$localreposdir_target" /etc/fstab ; then
+        echo 'force_drivers+="9pnet_virtio"' > /etc/dracut.conf.d/03-libvirt.conf
+        dracut -f
```

Comment:
I think, we only need to run this, if it was not there yet. dracut can take some minutes to run.

It seems we are running it in 3 places at: addupdaterepo prepareinstallcrowbar prepare_cloudupgrade_admin_repos_6_to_7